src/lib/y2storage/devicegraph.rb

```diff
@@ -166,6 +166,14 @@ def filesystems
       Filesystems::Base.all(self)
     end
 
+    # @param from [String] mountpoint of the filesystem (e.g. "/").
```

Comment:
I'm pretty sure yardoc would complain about the mismatch in the name of the argument. Have you checked?